### ISSUE TITLE
Module build failed: Error: ENOENT: no such file or directory /esm/polyfills/index.js

### ISSUE THREAD

**Issue by fabiofcs** [ionic-team/stencil#4588]
This issue is old but we are facing this again. I'm using version 4.0.1 (the latest at this moment)
```console
Module build failed: Error: ENOENT: no such file or directory, open '<PATH TO PROJECT>\dist\esm\polyfills\index.js'
```
There is no content in the polyfills folder:
![image](https://github.com/ionic-team/stencil/assets/121025845/f98a8a6b-62f0-42ef-be1a-809565f692e7)

_Originally posted by @fabiofcs in https://github.com/ionic-team/stencil/issues/1649#issuecomment-1639962545_

**Comment by rwaskiewicz** [comment]
Hey @fabiofcs 👋 

Thanks for the issue report!  Was this previously working for you? If so, what version of Stencil were you using?

Can you also do me a favor and create a minimal reproduction case for the team to take a look at?

Thanks in advance!

**Comment by fabiofcs** [comment]
Thanks for the support @rwaskiewicz. After troubleshooting my dependencies, the problem is with my `.lock` file and it's not related to Stencil.
I'll close the issue.

**Comment by VaggelisKa** [comment]
@fabiofcs What did you notice exactly? Im facing the exact same issues on Stencil 4.0.1

**Comment by fabiofcs** [comment]
> @fabiofcs What did you notice exactly? Im facing the exact same issues on Stencil 4.0.1

@VaggelisKa Yes, I'm using the same version `4.1.0`
actually, I have some points:
> Considering the polyfill folder path: `dist/esm/polyfills`

- I realize the `polyfill/` will be deleted when I run the Stencil the `watch` option: `stencil build --watch`.
- When I run this command: `stencil build --docs` the folder `polyfill/`  is not deleted.
- As I'm using [Storybook ](https://storybook.js.org/) to run my component library I need this polyfill to register the components, for example:
```ts
import { defineCustomElements } from '../loader'
...
defineCustomElements(); // This function will call the polyfill library
```
- To solve this problem, somehow I realized that the `polyfill/` is generated after some saved change (just to generate build again while watching).

- Do u think that issue should be "Reopen"?

**Comment by VaggelisKa** [comment]
@fabiofcs  Ok we are facing EXACTLY the same issue.  Im also using the `--watch` for storybook changes. Could you try adding the `--es5` flag on the build? So something like `stencil build --watch --es5`. When i do that i notice that the polyfills get removed for a split second and then added back

**Comment by VaggelisKa** [comment]
Perhaps we should re-open this. But probably we should try to provide some reproduction repo

**Comment by fabiofcs** [comment]
@VaggelisKa with the `--es5` seems to work fine. I mean,  the polyfills content is generated with the build process. 
However, without `--es5` option, the Polyfill is generated just only after saving some content to rebuild again regardless Storybook build (this is happening testing only the Stencil build).

@rwaskiewicz This is the expected behavior? The option: `--es5` should be a possible solution. If so, is there some docs to explain that?

**Comment by rwaskiewicz** [comment]
@fabiofcs I'm not sure if we have anything specific explaining how `polyfills` gets generated. To double check, is that what you're asking here?

> This is the expected behavior? The option: --es5 should be a possible solution. If so, is there some docs to explain that?

**Comment by DmitryAvanesov** [comment]
Facing this issue after updating to Stencil v4.1.0

**Comment by rwaskiewicz** [comment]
Please note, this issue was closed due to lack of reproduction case and is likely not being tracked by most of the Stencil team.

https://github.com/ionic-team/stencil/issues/4661 _may_ be helpful for some to follow, although without reproduction case(s), we can't entirely say for sure. If folks believe #4661 is not the same issue, please open a new issue with a minimal reproduction case.
